src/flight_planning.ts

```diff
@@ -97,13 +97,11 @@ export async function generateFlightPlan(
     console.error("No node in the flight plan to start from...");
     return flightPlan;
   }
-  // @ts-ignore
-  mapService.nodeToPointPromise = promisify<
-    NodeToPointRequest,
-    NodeToPointResponse
-  >(mapService.nodeToPoint);
-  // @ts-ignore
-  const response = await mapService.nodeToPointPromise(
+  const nodeToPointPromise = promisify<NodeToPointRequest, NodeToPointResponse>(
```

Comment:
I think I initially had it like this, but was thinking later on this promisify stuff could happen when the service is instantiated. Also fine to do it this way and we can move these functions to the services file, similar end result and no ts-ignore.